### PR TITLE
Why is this work important?

### LINKED ISSUE #314: Introduction is too deep, too soon
The introduction gets right into things without explaining why this problem is even worth solving.

### PR DESCRIPTION
**Pull request by elear**
Here's a strawman intro that explains the import of the work in a concrete manner.

Fixes #314

### CODE REVIEW

**Comment by henkbirkholz** [comment]
While I can see the use of the additional content, I thought you intended to improve the introduction. Do you really intended to start an abstract text with a question? I am not sure how people who actually use a CMS would react to that.

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -138,9 +138,24 @@ informative:
...
-In network protocol exchanges it is often the case that
-one entity requires believable evidence about the operational state of a remote
-peer.  Such evidence is typically conveyed as claims
+How should one system know that another system can be trusted?
+
+ * Wouldn't it be nice if one bank backend system had reason to
+   believe that another bank backend system was trustworthy?
+
+ * Wouldn't you like to know that electronic health records (EHRs)
+   were shared only among systems that were trustworthy?
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:147]: Second person is great, but are we doing it?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -138,9 +138,24 @@ informative:
 
 --- abstract
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:139]: This belongs in the intro.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -138,9 +138,24 @@ informative:
...
+How should one system know that another system can be trusted?
+
+ * Wouldn't it be nice if one bank backend system had reason to
+   believe that another bank backend system was trustworthy?
+
+ * Wouldn't you like to know that electronic health records (EHRs)
+   were shared only among systems that were trustworthy?
+
+How does one system determine whether another system is trustworthy?
+This is the goal of the Remote Attestation architecture (RATS).
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:150]: RATS is the acronym for Remote Attestation ProcedureS

**Review by mcr — CHANGES_REQUESTED** [review]

**Review by mcr — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -138,20 +138,35 @@ informative:
...
-In network protocol exchanges it is often the case that
-one entity requires believable evidence about the operational state of a remote
-peer.  Such evidence is typically conveyed as claims
-about the peer's software and hardware platform, and is subsequently appraised in order
-to assess the peer's trustworthiness.  The process of generating and
-appraising this kind of evidence is known as remote
-attestation.  This document describes an architecture for remote attestation
-procedures that generate, convey, and appraise evidence about a peer's
-operational state.
+In network protocol exchanges it is useful for one end of a
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:141]:
```suggestion
In network protocol exchanges it is often useful for one end of a
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -138,20 +138,35 @@ informative:
...
+communication to know if the other end is in a proper operating state.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:142]:
```suggestion
communication to know whether the other end is in a proper operating state.
```
(https://grammarist.com/grammar/if-vs-whether/ says "if" is considered informal, though the meaning is the same, so to me "whether" reads better)

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -138,20 +138,35 @@ informative:
...
+that make such tests possible through the process of generating,
+conveying, and evaluating evidentiary claims.  An attempt is made to
+provide for a model that is neutral toward processor architectures,
+the content of claims, and protocols.
 
 --- middle
 
 # Introduction
 
+How should one system know that another system can be trusted?  This
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:153]:
```suggestion
How can one system know that another system can be trusted?  This
```

**Review by dthaler — APPROVED** [review]
Ok, but see suggested wording improvements

File: draft-ietf-rats-architecture.md (hunk 5 above)
Comment (mcr) [inline draft-ietf-rats-architecture.md]:
```suggestion
communication to know if the other end is in an intended operating state.
```

File: draft-ietf-rats-architecture.md [hunk 7]
```diff
@@ -138,20 +138,35 @@ informative:
...
-one entity requires believable evidence about the operational state of a remote
-peer.  Such evidence is typically conveyed as claims
-about the peer's software and hardware platform, and is subsequently appraised in order
-to assess the peer's trustworthiness.  The process of generating and
-appraising this kind of evidence is known as remote
-attestation.  This document describes an architecture for remote attestation
-procedures that generate, convey, and appraise evidence about a peer's
-operational state.
+In network protocol exchanges it is often useful for one end of a
+communication to know whether the other end is in a proper operating state.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:142]:
```suggestion
communication to know whether the other end is in an intended operating state.
```

File: draft-ietf-rats-architecture.md [hunk 8]
```diff
@@ -138,20 +138,35 @@ informative:
...
+This memo provides an architectural overview of the entities involved
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:143]:
```suggestion
This document provides an architectural overview of the entities involved
```

File: draft-ietf-rats-architecture.md [hunk 9]
```diff
@@ -138,20 +138,35 @@ informative:
...
+provide for a model that is neutral toward processor architectures,
+the content of claims, and protocols.
 
 --- middle
 
 # Introduction
 
+How can one system know that another system can be trusted?  This
+age old question has new interest and relevance in a world where
+trusted computing elements are maturing in processor architectures.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:155]:
```suggestion
The question of how one system can know that another system can be trusted has found new interest and relevance in a world where trusted computing elements are maturing in processor architectures.
```

**Comment by mcr** [comment]
Closed with commit:d7227b2

**Pull request closed without merging**